packages/aws-cdk/lib/api/cxapp/cloud-assembly.ts

```diff
@@ -336,11 +336,11 @@ export class StackCollection {
     }
 
     if (errors && failAt != 'none') {
-      throw new AssemblyError('Found errors');
+      throw AssemblyError.fromStacks('Found errors', this.stackArtifacts);
```

Comment:
Suggestion to rename to `withStacks` and `withCause` instead, but your call.

Reply:
I had that before! Happy to change.